python/lsst/cp/pipe/ptc/cpSolvePtcTask.py

```diff
@@ -432,16 +404,7 @@ def _getInitialGoodPoints(means, variances, maxDeviationPositive, maxDeviationNe
             Input array with mean signal values.
         variances : `numpy.array`
             Input array with variances at each mean value.
-        maxDeviationPositive : `float`
-            Maximum deviation from being constant for the variance/mean
-            ratio, in the positive direction.
-        maxDeviationNegative : `float`
-            Maximum deviation from being constant for the variance/mean
-            ratio, in the negative direction.
-        minMeanRatioTest : `float`
-            Minimum signal value (in ADU) after which to start examining
-            the ratios var/mean.
-       minVarPivotSearch : `float`
+        minVarPivotSearch : `float`
             Minimum variance point (in ADU^2) after which the pivot point
             wher the variance starts decreasing should be sought.
```

Comment:
`wher` → `where`. But also that sentence is just wrong somehow actually... Maybe:

`The variance (in ADU^2), above which, the point of decreasing variance should be sought.`